operator-common/src/main/java/io/strimzi/operator/common/model/Ca.java

```diff
@@ -595,6 +595,7 @@ public void createRenewOrReplace(String namespace, String clusterName, Map<Strin
         // cluster CA certificate annotation handling
         Map<String, String> certAnnotations = new HashMap<>(2);
         certAnnotations.put(ANNO_STRIMZI_IO_CA_CERT_GENERATION, String.valueOf(caCertGeneration));
+        certAnnotations.put(Annotations.ANNO_STRIMZI_IO_FORCE_RENEW, "false");
```

Comment:
So, for this and all the other annotations which the operator owns through this -> I guess you would also need to change the commands in the docs to force the update? Or will they work as they are today?